fcrepo-auth-webac/src/test/java/org/fcrepo/integration/auth/webac/WebACRecipesIT.java

```diff
@@ -2084,70 +2081,6 @@ public void testBinaryAndDescriptionAllowed() throws Exception {
 
     }
 
-    @Test
-    public void testRequestWithEmptyPath() throws Exception {
```

Comment:
This test is removed because it appears that as a result of one of the upgrades, likely Jersey, empty path segments, `//`, are normalized, so `parent/child` and `parent//child` are indistinguishable to the application.

Reply:
It appears that HttpClient changed in the 4.5.7 release to remove empty paths by default. Can we leave this test in-place, replacing the last portion with:
```
        // Ensure HttpClient does not remove empty paths
        final RequestConfig config = RequestConfig.custom().setNormalizeUri(false).build();

        // Admin bypasses ACL resolution gets 409.
        final HttpGet getAdminRequest = getObjMethod(parent + "//" + child);
        setAuth(getAdminRequest, "fedoraAdmin");
        getAdminRequest.setConfig(config);
        assertEquals(BAD_REQUEST.getStatusCode(), getStatus(getAdminRequest));
        // User
        final HttpGet getUserRequest = getObjMethod(parent + "//" + child);
        getUserRequest.setConfig(config);
        setAuth(getUserRequest, username);
        assertEquals(BAD_REQUEST.getStatusCode(), getStatus(getUserRequest));
```

As a note, I tried this config update at the `AbstractResourceIT.getObjMethod(id)` level, but it breaks many of the other tests in `WebACRecipesIT` that include double slashes in their test fixtures.

Reply:
Nice! Thanks for figuring that out.